src/linagora.esn.controlcenter/app/common/application-menu.directive.js

```diff
@@ -2,10 +2,19 @@
 
 angular.module('linagora.esn.controlcenter')
 
-.directive('controlcenterApplicationMenu', function(applicationMenuTemplateBuilder) {
-  return {
-    retrict: 'E',
-    replace: true,
-    template: applicationMenuTemplateBuilder('/#/controlcenter', { url: '/controlcenter/images/control-center-icon.svg' }, 'Control Center')
-  };
-});
+  .directive('controlcenterApplicationMenu', function(applicationMenuTemplateBuilder) {
+    return {
+      retrict: 'E',
+      replace: true,
+      template: applicationMenuTemplateBuilder('/#/controlcenter', { url: '/controlcenter/images/control-center-icon.svg' }, 'Control Center')
+    };
+  })
+
+  .directive('controlcenterMenuAccount', function(controlCenterMenuTemplateBuilder, CONTROL_CENTER_MENU_ACCOUNT) {
```

Comment:
Where does this CONTROL_CENTER_MENU_ACCOUNT constant come from ?

Reply:
from [here](https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/blob/main/src/modules/linagora.esn.account/frontend/js/constants.js#L36)